generator/metaline_builder.hpp

```diff
@@ -0,0 +1,31 @@
+#pragma once
```

Comment:
The rule of thumb: the name of the class must be the same as the name of the containing file. Of course, there may be exceptions.